input/tar2/tar_input.go

```diff
@@ -0,0 +1,163 @@
+package tar2
+
+import (
+	"archive/tar"
+	"bytes"
+	"crypto/sha512"
+	"encoding/base64"
+	"hash"
+	"io"
+	"os"
+	"path"
```

Comment:
I don't think `path.Clean` will handle that at all; none of its steps mention anything about different separators.
But that might be academic since in practice, we only really operate on one architecture.

I was actually thinking about just commenting the *import*, not each *usage*. Would you be amicable to that?